packages/react/src/utils/use-media-query/index.ts

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+
+function useMediaQuery(query: string, initialValue = false): boolean {
```

Comment:
Took the `useMediaQuery` hook from Walnut, my thought being the hook could be used elsewhere in Cauldron/exported, then imported into Walnut (it's already a common module).